langfuse/model.py

```diff
@@ -72,3 +73,12 @@ def __eq__(self, other):
             )
 
         return False
+
+    def get_langchain_prompt(self) -> str:
+        """Converts string of Langfuse prompt template prompt into string compatible
+        with Lanchain PromptTemplate.
```

Comment:
nit: type lanchain -> langchain

Reply:
thx, will merge to wip branch and fix this